engine/api/v2_config.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+import (
+	"context"
+	"github.com/ovh/cds/sdk"
+	"net/http"
+
+	"github.com/ovh/cds/engine/service"
+)
+
+// configVCSGPGKeysHandler return all gpg public keys for all vcs server
+func (api *API) configVCSGPGKeysHandler() ([]service.RbacChecker, service.Handler) {
+	return nil,
```

Comment:
no service.RBAC needed?

Reply:
It's to get public gpg key like the github one, so no